lib/plugins/aws/customResources/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const path = require('path');
+const createZipFile = require('../../../utils/fs/createZipFile');
+
+function addCustomResourceToService(resourceName) {
+  let FunctionName;
+  let Handler;
+
+  let customResourceFunctionLogicalId;
+  const srcDirPath = path.join(__dirname, 'resources');
+  // TODO: not sure if this is deterministic given that we have packaging options
+  // on the service and function level
```

Comment:
@medikoo do you know the answer to this question?

Reply:
@medikoo you were right. I just tested it with a custom artifact and it works just fine :+1: